.azure-pipelines-templates/model_checking.yml

```diff
@@ -19,15 +19,15 @@ jobs:
           python3 ./tla/install_deps.py
         displayName: Setup
 
-      - script: ./tlc.sh -workers auto consensus/MCccfraft.tla -dumpTrace tla MCccfraft.trace.tla -dumpTrace json MCccfraft.json
+      - script: ./tlc.sh -workers auto consensus/MCccfraft.tla -dumpTrace tla MCccfraft.trace.tla -dumpTrace json MCccfraft.json -coverage 10
         workingDirectory: tla
         displayName: MCccfraft.cfg
 
-      - script: ./tlc.sh -workers auto -config consensus/MCccfraftAtomicReconfig.cfg consensus/MCccfraft.tla -dumpTrace tla MCccfraftAtomicReconfig.trace.tla -dumpTrace json MCccfraftAtomicReconfig.json
+      - script: ./tlc.sh -workers auto -config consensus/MCccfraftAtomicReconfig.cfg consensus/MCccfraft.tla -dumpTrace tla MCccfraftAtomicReconfig.trace.tla -dumpTrace json MCccfraftAtomicReconfig.json -coverage 10
         workingDirectory: tla
         displayName: MCccfraftAtomicReconfig.cfg
 
-      - script: ./tlc.sh -workers auto -config consensus/MCccfraftWithReconfig.cfg consensus/MCccfraft.tla -dumpTrace tla MCccfraftWithReconfig.trace.tla -dumpTrace json MCccfraftWithReconfig.json
+      - script: ./tlc.sh -workers auto -config consensus/MCccfraftWithReconfig.cfg consensus/MCccfraft.tla -dumpTrace tla MCccfraftWithReconfig.trace.tla -dumpTrace json MCccfraftWithReconfig.json -coverage 10
```

Comment:
Only the last one takes more than 10 minutes:

![image](https://github.com/microsoft/CCF/assets/4016369/6f40d8f7-5b08-496b-93b5-133405d7366d)

I think we either want coverage for the whole run (in which case I guess it's `-coverage 30`), or if we want something more detailed with trends, `-coverage 1`.